src/libLocalDescriptorCache/DescDbProxy.cpp

```diff
@@ -301,10 +301,8 @@ std::string DescDbProxy::prepareCacheFile()
     return dbFilePath;
 }
 
-void DescDbProxy::getDescriptors(const std::string &f_hostAddress)
-{  
-    std::string cacheFilePath = prepareCacheFile();
-
+static bool getDbFileFromCacheFile(const std::string& cacheFilePath, localDescDb::DescriptorDb& file)
```

Comment:
```suggestion
/// returns false if file was found but could not be parsed. true otherwise.
static bool getDbFileFromCacheFile(const std::string& cacheFilePath, localDescDb::DescriptorDb& file)
```

Reply:
done